ONSdigital/dp-identity-api
# Feature/set group members endpoint

### What

Set update users in group from list of users

### How to review

1) code standards gofmt -l . does produce some format issues but not on the files in this release
2) make test pass and coverage has increased
3) make test-coverage pass and coverage has increased
4) check compliant functionality 

### Who can review

anyone but me
